### PR TITLE
Home page QA

### LINKED ISSUE #107: [FE] home page contribution list
image

contribution list should also apply the same width as the rest on the larger screen.

### LINKED ISSUE #110: [FE] Mobile home UI, bug
image

- On the mobile, user can trigger a manual video playback. I think there is a way to let user not interact with the background video.

- contribution list can be wider, the padding left and right is too large

### PR DESCRIPTION
**Pull request by CJ-Rose**
- fixed contribution table to same width as FAQ for large screens
- made mobile contribution cards wider
- changed bg from video to static image of portal, i think it's cool idk do you like it @ChialiT ?

~https://d4fba36f.unirep-trusted-setup.pages.dev/~

### CODE REVIEW

**Comment by ChialiT** [comment]
image
I think this works, the image seems a bit too big, because i could drag the screen to left or right side. can try overflow:hidden or make the image smaller?

otherwise we could merge this.

File: packages/frontend/src/pages/Home.jsx [hunk 1]
```diff
@@ -18,12 +18,21 @@ export default observer(() => {
   }
   return (
     <>
-      <video autoPlay muted loop poster playsInLine>
-        <source
-          src={require('../../public/unirep-ceremony-hero.mp4')}
-          type="video/mp4"
-        />
-      </video>
+      {ui.isMobile ? (
```
Comment (ChialiT) [inline packages/frontend/src/pages/Home.jsx:21]: this is good

**Review by ChialiT — APPROVED** [review]
just a minor suggestion, otherwise lets merge it.

File: packages/frontend/src/pages/Home.jsx (hunk 1 above)
Comment (CJ-Rose) [reply packages/frontend/src/pages/Home.jsx:21]: hmm idk i didn't see that in the mobile simulator, yeah it's kinda wonky on the phone. i'll try to fix it first.

**Comment by CJ-Rose** [comment]
still fixing: 
- static portal background
- bug: circuit selection text blue on mobile
- contribution cards padding corrections on stats page

**Pull request merged**